src/GsvdInitialization.jl

```diff
@@ -117,7 +117,7 @@ function gsvdrecover(X::AbstractArray, W0::AbstractArray, H0::AbstractArray, kad
         U0, S0, V0 = f
         U0, S0, V0 = U0[:,1:n], S0[1:n], V0[:,1:n]
         Hadd, Λ = init_H(U0, S0, V0, W0, H0, kadd)
-        Wadd, a = init_W(X, W0, H0, Hadd)
+        Wadd, a = init_W(X, W0, H0, Hadd)    
```

Comment:
trailing whitespace. You can set vscode to delete it automatically: https://www.codexcafe.com/blog/remove-trailing-whitespace/